src/components/login.jsx

```diff
@@ -0,0 +1,130 @@
+import React from "react";
+
+const Signup = () => {
```

Comment:
this ought to be a login component?? isnt it? pls rectify this

Reply:
Yes, it is. Sorry for the inconvenience. I forgot to change the 'const' after copying and pasting from signup.

Reply:
Please ignore the App.jsx file

Reply:
It seems you've not reviewed the request. I can see it's pending.
![Screenshot (110)](https://github.com/Stemmanuel95/IdealClass_frontend/assets/9413367/73e0fda9-83de-4da2-930f-53c1bfc451c6)